content/influxdb/cloud-serverless/reference/syntax/delete-predicate.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Delete predicate syntax
+list_title: Delete predicate
+description: >
+  InfluxDB uses an InfluxQL-like predicate syntax to determine what data points to delete.
+menu:
+  influxdb_cloud_serverless:
+    parent: Syntax
+    name: Delete predicate
+weight: 104
+influxdb/cloud-serverless/tags: [syntax, delete]
+related:
+  - /influxdb/cloud-serverless/write-data/delete-data/
+  - /influxdb/cloud-serverless/reference/cli/influx/delete/
+---
+{{% warn %}}
+#### InfluxDB Cloud Serverless does not support data deletion
+
+InfluxDB Cloud Serverless does not currently support deleting data.
+This command is only supported when used with **InfluxDB OSS v2** and
+**InfluxDB Cloud (TSM)**.
+{{% /warn %}}
+
+{{< duplicate-oss >}}
+
+{{% warn %}}
+#### InfluxDB Cloud Serverless does not support data deletion
+
+InfluxDB Cloud Serverless does not currently support deleting data.
+This command is only supported when used with **InfluxDB OSS v2** and
+**InfluxDB Cloud (TSM)**.
+{{% /warn %}}
```

Comment:
We could probably just remove all the content and the duplicate warning at the end of the doc.

```suggestion
```